src/virtualenv/create/via_global_ref/builtin/cpython/mac_os.py

```diff
@@ -102,6 +103,15 @@ def reload_code(self):
         )
         return result
 
+    @classmethod
+    def can_create(cls, interpreter):
+        if IS_MAC_ARM64:
+            # https://github.com/pypa/virtualenv/issues/2024
+            logging.fatal("Creating virtualenv for Python 2 on Apple Silicon Macs is not supported as of now.")
```

Comment:
You'll need to remove this, as this would still print if someone adds support via a plugin

Reply:
Thanks for a review @gaborbernat !

Unfortunately I am not familiar with this project's structure yet. Can you recommend a better place to put this check then?

Alternatively we could change this to:
```
logging.warning("The built-in creator does not handle creating virtualenvs for Python 2 on Apple Silicon Macs as of now.")
```
- that should not be misleading for a person that adds their own plugin to support this case.

Reply:
The location is good. There isn't a place to put the warning message though, so that needs to be removed just.

Reply:
I am afraid that without this message this PR doesn't make sense as the whole point was to show a friendly error that explains what happened.

The generic "No virtualenv implementation" error would make look for issues in my machine Python setup, not assume that this is actually a missing virtualenv feature...

Maybe it would be easier for you to implement the missing plugin instead? I would love to do it myself but I just don't know enough about Mach-O hacking to it to make sense... I am willing to do all the testing you need and as it seems that we are in similar timezones (BST / CEST) we could sync easily.

Reply:
> I would love to do it myself but I just don't know enough about Mach-O hacking to it to make sense

Me neither... And I don't have access to an Apple M1 machine to write an implementation for it or time to do that. So guess we wait until someone comes along who does.